docs/cloud/managing-airbyte-cloud.md

```diff
@@ -312,13 +312,42 @@ Changes to data residency will not affect any sync in progress.
 
 :::
 
-## Buy credits
+## Manage credits
+
+### Enroll in the Free Connector Program
+
+By enrolling the Free Connector Program, you can sync connections with [alpha](https://docs.airbyte.com/project-overview/product-release-stages#alpha) or [beta](https://docs.airbyte.com/project-overview/product-release-stages/#beta) connectors at no cost.
+
+Before enrolling in the program, [set up](https://docs.airbyte.com/cloud/getting-started-with-airbyte-cloud#set-up-a-source) at least one alpha or beta connector and verify your email, if prompted.
```

Comment:
Why "if prompted"? Either they should be prompted or they won't. What are we trying to say here?

Reply:
I said "if prompted" here because users who have not verified their email will see "verify your email" in the UI, and they will need to do that before they can enroll. Users who have already verified their email will not see "verify your email." Alternatively, we could say something like "and make sure your email is verified." They can't miss the verify email step because they will not be able to enroll without first verifying their email. 
<img width="892" alt="Screenshot 2023-01-25 at 2 37 47 PM" src="https://user-images.githubusercontent.com/106352739/214707785-d32dc2ed-a153-49d6-bb40-b414510e82d9.png">
<img width="689" alt="Screenshot 2023-01-25 at 2 39 56 PM" src="https://user-images.githubusercontent.com/106352739/214708049-f04e5fa8-25ad-4064-850b-eb737f0caa8b.png">

Reply:
Ahh, in that case, a better way to phrase it is "verify your email if you haven't already"